filter/etc/httpd/admin-conf.d/rspamd.conf

```diff
@@ -17,6 +17,7 @@
     AuthUserFile /etc/httpd/admin-conf/rspamd.secret
     AuthPAMService rspamd
     Require valid-user
+    Options SymLinksIfOwnerMatch
```

Comment:
As the error happens when `/rspamd/actions` is requested, I'd expect the directive is added under line L25. This stanza does not use mod_rewrite directives :thinking:

Reply:
we could move it to L27, it works also I think